packages/schematics/angular/migrations/update-6/index.ts

```diff
@@ -431,6 +432,7 @@ function extractProjectsConfig(
       // Serve target
       const serveOptions: JsonObject = {
         browserTarget: `${name}:build`,
+        ...(app.baseHref ? { baseHref: app.baseHref } : {}),
```

Comment:
It's not needed in the serve, it's just an override for the browser target there. Other than that LGTM.

Reply:
Good catch, updated!